tests/containers/registry.pm

```diff
@@ -37,31 +38,31 @@ sub registry_push_pull {
     die 'Argument $runtime not provided!' unless $runtime;
 
     # Pull $image
-    assert_script_run "$runtime pull $image",            600;
-    assert_script_run "$runtime images | grep '$image'", 60;
+    assert_script_run $runtime->runtime . " pull $image",            600;
```

Comment:
Why don't you use here the new functions `$runtime->pull` ??

Reply:
there are many parts that i havent touch. i went with blocks of code or per functions. i have changed anything in this module other than port the variable to the object instance

Reply:
so, it's something to do after this PR, right?

Reply:
correct. there are many improvements to follow up. next step will be to unify the _docker _podman. I believe that it is also more easy to identify the changes that they need to be done. step by step